unity/Assets/Scripts/ProceduralTools.cs

```diff
@@ -1874,36 +1876,37 @@ Collider collider
             }
 
             var lightingRoot = new GameObject(DefaultLightingRootName);
+            if (house.proceduralParameters.lights != null) {
```

Comment:
Should we throw an error if `house.proceduralParameters.lights == null`? Seems like this should always be a (possibly empty) list.

Reply:
A house with no lights is a valid house, some tests use this and there is global lighting even without point lights